client/settings/payment-gateway-section/__tests__/index.test.js

```diff
@@ -14,6 +14,7 @@ jest.mock( '@woocommerce/navigation', () => ( {
 
 jest.mock( '../../../data/account/hooks', () => ( {
 	useAccount: jest.fn().mockReturnValue( { data: {} } ),
+	useGetCapabilities: jest.fn().mockReturnValue( {} ),
```

Comment:
Maybe we can add a test here to simulate a case where the capability status pill would be rendered?

Reply:
Sounds good. Added new test in c731f84.